configfiles/graphs/test-tiny.graphml

```diff
@@ -29,5 +30,17 @@
       <data key="active">0</data>
       <data key="type">EXC</data>
     </node>
+    <edge id="e1" source="0" target="1">
+      <data key="weight">0.5</data>
+    </edge>
+    <edge id="e2" source="1" target="2">
+      <data key="weight">3.2</data>
+    </edge>
+    <edge id="e3" source="2" target="3">
+      <data key="weight">3.3</data>
+    </edge>
+    <edge id="e4" source="3" target="0">
+      <data key="weight">6.4</data>
+    </edge>
```

Comment:
Let's make sure the weights are within "reasonable" ranges from the code (particularly, AllNeuroEdges), and that they are consistent in sign with the source neuron type. So:

- **e1**: 5.0e-9
- **e2**: -3.2e-8
- **e3**: 3.3e-8
- **e4**: 6.4e-8